Doc/reference/expressions.rst

```diff
@@ -1156,7 +1156,8 @@ a user-defined function:
    first thing the code block will do is bind the formal parameters to the
    arguments; this is described in section :ref:`function`.  When the code block
    executes a :keyword:`return` statement, this specifies the return value of the
-   function call.
+   function call.  If execution falls off the end of the code block without
```

Comment:
```suggestion
   function call.  If execution reaches the end of the code block without
```

Reply:
I used “falls off the end” because it's the phrasing that's already used in [Generator functions](https://docs.python.org/3/reference/datamodel.html#generator-functions) et al. But you should merge whichever wording you prefer.

Reply:
As non-native English speaker, I would say that the proposed change is close to "simple" English; foreigners could understand it a little easier.

Reply:
@asvetlov Please merge as makes sense. I'm fine wih either option. Thanks @jmarshall for the additional context and PR too.

Reply:
Eah, multi-word verbs cannot be understood by translating all the words separately :)
Modern automated tools like Google Translate does this job well but still...